src/lib/php/Dao/ClearingDao.php

```diff
@@ -732,29 +741,39 @@ function getClearedLicenseIdAndMultiplicities(ItemTreeBounds $itemTreeBounds, $g
    * @param int $groupId
    * @param int $userId
    */
-  public function markDirectoryAsIrrelevant(ItemTreeBounds $itemTreeBounds,$groupId,$userId)
+  public function markDirectoryAsDecisionType(ItemTreeBounds $itemTreeBounds, $groupId, $userId, $decisionMark)
   {
-    $this->markDirectoryAsIrrelevantIfScannerDetected($itemTreeBounds, $groupId, $userId);
-    $this->markDirectoryAsIrrelevantIfUserEdited($itemTreeBounds, $groupId, $userId);
+    if ($decisionMark == "doNotUse") {
```

Comment:
Why not pass the internal type (`DecisionTypes::DO_NOT_USE`) instead of strings?